spec/dummy_driver_spec.cr

```diff
@@ -138,6 +138,12 @@ describe DummyDriver do
         end
       end
 
+      it "with as named tuple" do
```

Comment:
"with a named tuple" is enough, the `as` is implicit.

Reply:
ok